### PR TITLE
make shell script that do 'cargo run' and launch browser(google-chrome)

### PR DESCRIPTION
**Pull request by seiay**
## Description
I made a shell script that can do a 'cargo run'  command and launch a browser(google-chrome).
You can start up the web app only by executing start-browser. sh.

## Test performed
I confirmed the start-up of the web app by `./start-browser.sh`.

### CODE REVIEW

File: start-browser.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/zsh
+
+# cargo run
+cargo build
+(
+	sleep 1
```
Comment (asana17) [inline start-browser.sh]: This expression (`sleep 1`) is OK temporarily, but we need to check target app is running by some method (for example, the `ps` command). You don't need to change it in this PR.

**Review by asana17 — CHANGES_REQUESTED** [review]
The script name `start-browser.sh` looks like starting only the browser. Something like `run_with_browser.zsh`is better,  I think.

File: start-browser.sh (hunk 1 above)
Comment (seiay) [reply start-browser.sh]: Thanks!

> but we need to check target app is running by some method (for example, the ps command). 

Ok.

**Comment by seiay** [comment]
> The script name start-browser.sh looks like starting only the browser. Something like run_with_browser.zshis better, I think.

I think so too.
I renamed the script name and also changed README.

**Review by asana17 — APPROVED** [review]

**Pull request merged**